modules/reference/pages/rpk/rpk-container/rpk-container-start.adoc

```diff
@@ -2,57 +2,95 @@
 
 Start a local container cluster.
 
+This command utilizes Docker to initiate a local container cluster. Use the `--nodes` (or the shorthand version `-n`) flag to specify the number of brokers.
+
+The initial broker starts on default ports, with subsequent brokers' ports offset by 1000. You can use the following flags to specify listeners ports:
+
+* --kafka-ports
+
+* --admin-ports
+
+* --rpc-ports
+
+* --schema-registry-ports
+
+* --proxy-ports
+
+Each flag accepts a comma-separated list of ports for your listeners. Use the `--any-port` flag to let `rpk` randomly select an available port on the host machine.
+
+In case of IP address pool conflict, you may specify a custom subnet and gateway using the `--subnet' and `--gateway` flags respectively.
+
+By default, this command uses `redpandadata/redpanda:latest` Redpanda container image. You can specify a container image by using the `--image` flag. See the available images at xref:https://hub.docker.com/r/redpandadata/redpanda/tags[Docker Hub].
+
 == Usage
 
 [,bash]
 ----
 rpk container start [flags]
 ----
 
-== Example
+== Examples
 
-If you're following xref:get-started:quick-start.adoc#tabs-1-three-brokers[QuickStart -  Deploy Redpanda to Docker with Three Nodes], you can run `rpk container start -n 3` to start your containers:
+Start a 3-broker cluster:
 
-[,bash]
-----
-$ rpk container start -n 3
-Starting cluster
-Waiting for the cluster to be ready...
-Cluster started!
-  NODE-ID  STATUS   KAFKA-ADDRESS    ADMIN-ADDRESS    PROXY-ADDRESS    
-  0        running  127.0.0.1:45841  127.0.0.1:40705  127.0.0.1:36875  
-  1        running  127.0.0.1:42467  127.0.0.1:38959  127.0.0.1:37665  
-  2        running  127.0.0.1:44941  127.0.0.1:46047  127.0.0.1:45783  
+```bash
+rpk container start -n 3
+```
 
-You can use rpk to interact with this cluster. E.g:
+Start a 1-broker cluster, selecting random ports for every listener:
 
-	rpk cluster info --brokers 127.0.0.1:45841,127.0.0.1:42467,127.0.0.1:44941
+```bash
+rpk container start --any-port
+```
 
-You may also set an environment variable with the comma-separated list of
-broker addresses:
+Start a 3-broker cluster, selecting the seed kafka and console port only:
 
-	export REDPANDA_BROKERS="127.0.0.1:45841,127.0.0.1:42467,127.0.0.1:44941"
-	rpk cluster info
-----
+```bash
+rpk container start --kafka-ports 9092 --console-port 8080
+```
+
+Start a 3-broker cluster, selecting every admin API port:
```

Comment:
```suggestion
Start a three-broker cluster, specifying the Admin API port for each broker:
```
I think this is what "every admin API port" is saying, feel free to ignore if I understood wrong!